pymeasure/instruments/aimtti/ld400p.py

```diff
@@ -0,0 +1,126 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2024 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.generic_types import SCPIMixin
+from pymeasure.instruments.validators import strict_discrete_set
+
+
+class LD400P(SCPIMixin, Instrument):
+    """Interface for the LD400P electronic DC load from Aim-TTi.
+
+    Note: the LD400 edition doesn't have digital interfacing, only the LD400P does.
+
+    The interface is described by the LD400[P] `manual`_.
+
+    .. _manual: https://resources.aimtti.com/manuals/
+                LD400+LD400P_Instruction_Manual_EN_48511-1730_8.pdf
+    """
+
+    unit_suffixes = ["A", "V", "W", "OHM", "SIE", "HZ"]
+
+    @classmethod
+    def remove_unit_suffix(cls, msg: str) -> str:
+        """Remove known unit suffixes from a msg.
+
+        The units of some replies depend on the mode of the device, hence all known
+        units are removed.
+        """
+        for suffix in cls.unit_suffixes:
+            msg = msg.replace(suffix, "")
+        return msg
+
+    def __init__(self, adapter, name="LD400", **kwargs):
+        super().__init__(adapter, name, **kwargs)
+
+    mode = Instrument.control(
+        "MODE?",
+        "MODE %s",
+        """Control the mode of the digital load. Can be one of:
+        ``("C", "P", "R", "G", "V")``, corresponding to (respectively) constant current,
+        power, resistance, conductance or voltage (string).
```

Comment:
```suggestion
        power, resistance, conductance, or voltage (string).
```
In English, you have a comma even before the "or" or "and" in an enumeration.